py/server/tests/test_table.py

```diff
@@ -859,6 +859,33 @@ def test_tree(self):
         tree_table = self.test_table.tail(10).tree(id_col='a', parent_col='c', promote_orphans=True)
         self.assertIsNotNone(tree_table)
 
+    def test_grouped_column_as_arg(self):
```

Comment:
I would add another set of test tables using java arrays instead of DB arrays.  e.g. `"Person = new Object[]{...}"` and `"Person = new String[]{...}"` etc.